src/routes/dashboard/user/application/+page.svelte

```diff
@@ -0,0 +1,253 @@
+            </Paragraph>
+            <div class="dashboard-user-application-button-wrapper">
+                <StyledLink title="Klicke hier, um zur Hauptseite zurück zu kehren"
+                            text="Hauptseite"
+                            href="/"
+                            newTab={false}
+                />
+            </div>
+        </div>
+    {:else if data.error}
+        <div class="dashboard-user-application-error">
+            <Paragraph classes="paragraph-pre-wrap"
+                       --text-align="center">{data.error.error}</Paragraph>
+            <div class="dashboard-user-application-button-wrapper">
+                <StyledLink title="Klicke hier, um zur Hauptseite zurück zu kehren"
+                            text="Hauptseite"
+                            href="/"
+                            newTab={false}
+                />
+                {#if data.roles.is_speaker}
+                    <StyledLink title={MenuItem.speakerEvents.description}
+                                text={MenuItem.speakerEvents.name}
+                                href={MenuItem.speakerEvents.url}
+                                newTab={false}
+                    />
+                    <StyledLink title={MenuItem.speakerSocialMedia.description}
+                                text={MenuItem.speakerSocialMedia.name}
+                                href={MenuItem.speakerSocialMedia.url}
+                                newTab={false}
+                    />
+                {/if}
+            </div>
+        </div>
+    {:else if data.data}
+        <form class="dashboard-user-application-form"
+              on:submit|preventDefault={trySave}>
+            <div class="dashboard-user-application-event-infos">
+                <HeadlineH2>{data.data.event.title}</HeadlineH2>
+                <TextLine>{formatDate(data.data.event.start_date, '%DD.%MM.')}
+                    - {formatDate(data.data.event.end_date, '%DD.%MM.%YYYY')}</TextLine>
+                <TextLine>Noch bis: {formatDate(data.data.event.call_for_papers_end, '%DD.%MM.%YYYY')}</TextLine>
+            </div>
+            <div class="dashboard-user-application-form-messages-wrapper">
+                <SaveMessage bind:this={saveMessage} />
+                {#each messages as message}
+                    <Message {message} />
+                {/each}
+            </div>
+            <div class="dashboard-user-application-section">
+                <HeadlineH2>Deine Bewerbung</HeadlineH2>
+                <SpeakerTeamMemberEventForm bind:this={eventForm}
+                                            bind:data={data.data.speaker}
+                                            displaySaveButton={false} />
+            </div>
+            <div class="dashboard-user-application-section">
+                <HeadlineH2>Social Media-Daten</HeadlineH2>
```

Comment:
```suggestion
                <HeadlineH2>Social-Media-Daten</HeadlineH2>
```
Or we just say "Social Media" (then we don't need the dash).